### ISSUE TITLE
How to type a Date object?

### ISSUE THREAD

**Issue by mlandauer** [pelotom/runtypes#29]
First off thanks for a really useful library. I've already used it very successfully on a small project.

However, now I've hit a problem with typing a Date object. I'm probably missing something fundamental here.

With the following:
```
import { Number, Record, InstanceOf, Static } from 'runtypes'

const Coords = Record({
  latitude: Number,
  longitude: Number,
  time: InstanceOf(Date)
})

type Coords = Static<typeof Coords>

let data: Coords = {
  latitude: 1.0,
  longitude: 1.0,
  time: new Date(2017, 0, 1, 10, 0, 0)
}
```

I get the type error:
```
...
Types of property 'time' are incompatible.
        Type 'Date' is not assignable to type 'DateConstructor'.
```

What am I missing?

if this is not the correct way to type a Date object, what is?

**Comment by pelotom** [comment]
@mlandauer Looks like this was a bug with `InstanceOf` which should be fixed now in `0.13.1`, give it a try.

**Comment by mlandauer** [comment]
@pelotom that works perfectly now. Thank you so much!
